bin/lamassu-migrate

```diff
@@ -23,10 +23,21 @@ const getMigrateFile = () => {
 
 db.none(createMigration)
   .then(() => Promise.all([db.oneOrNone(select), getMigrateFile()]))
+  .then(([qResult, migrateFile]) => {
+
+    process.env.SKIP_SERVER_LOGS = !(qResult && qResult.data.migrations.find(({ title }) => title === '1572524820075-server-support-logs.js'))
+
+    return [qResult, migrateFile]
+  })
   .then(([qResult, migrateFile]) => {
     if (!qResult && migrateFile) {
       return db.none('insert into migrations (id, data) values (1, $1)', [migrateFile])
     }
+  })
+  .then(() => {
+
+
+
```

Comment:
Remove this empty promise.

Reply:
lol, shame on me!

done, empty promise removed, and actually no additional promise was needed so I merged the other two into a single one.